supplementary_style_guide/style_guidelines/statements-about-the-future.adoc

```diff
@@ -0,0 +1,25 @@
+[[statements-about-the-future]]
+= Statements about the future
+
+[[making-statements-about-future-releases-or-plans]]
+== Making statements about future releases or plans
+
+Regarding future product releases, the _IBM Style Guide_ states, "Avoid claims that are related to the content or timing of a future product or release."  
+
+However, under some limited circumstances (such as _Release Notes_), circumstances will dictate that you refer to a future release, plan, or event. 
+In these situations, follow these guidelines:
+
+- When discussing future plans, use words such as "anticipate," "expect" or "plan."
```

Comment:
Can you put the period and the comma outside of the ending quote? That's a new ISG rule

Reply:
done